assets/scss/modules/_forms.scss

```diff
@@ -321,7 +321,7 @@
     /* Messages and confirmations the forms */
     .dp-message,
     .wrapper-errors {
-      display: inline-block;
+      display: block;
```

Comment:
Are you sure this change won´t break other validations? it seems widely used to me... 🤔